composer.json

```diff
@@ -15,12 +15,13 @@
         "keboola/php-temp": "^1.0",
         "guzzlehttp/oauth-subscriber": "^0.2",
         "guzzlehttp/cache-subscriber": "dev-master#aaab195",
-        "guzzlehttp/retry-subscriber": "dev-master#270393c"
+        "guzzlehttp/retry-subscriber": "dev-master#270393c",
+        "symfony/validator": "^5.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.2",
+        "phpunit/phpunit": "^8.5",
         "squizlabs/php_codesniffer": "^3.0",
-        "phpstan/phpstan": "^0.7"
+        "phpstan/phpstan": "^0.8"
```

Comment:
nejde tam dat rovnou posledni, nebo to je hodne zmen?

Reply:
neskusal som to, ale zo skusenosti s inymi komponentami je to vela zmien ... a kedze musim dorucit nejake veci pre CSAS, tak som si nemohol tomu dovolit venovat viac casu.